### DISCUSSION TITLE
How to receive webhooks from a public channel?

### DISCUSSION THREAD

**Discussion by AlexanderFalkenberg** [def-studio/telegraph#656] [Q&A]
I successfully created a bot, that receives webhooks, if I chat with the bot directly. 

I also want to receive webhooks and handle webhooks from other channels. For example a public channel.

- I added the boot as an Admin to the channel. 
- Set allow message from unknown chats to true and added the chat id to the database.
- I set the privacy to disabled.

Did I miss something?

**Comment by AlexanderFalkenberg** [comment]
Receiving Webhooks from a group works fine.

**Reply by fabio-ivona** [reply]
have you tried to log in order to check if any connection is incoming?

**Comment by D0nVitalio** [comment]
@fabio-ivona Ciao! I managed to atach bot to channel and i see in table `telegraph_chats` `id 2` , `chat_id -10000000001` , `name [channel] abc123`
Next, i have 
`protected function handleChatMessage(Stringable $text): void{ $this->reply('TestReplyToAnyText'); }`
and if I type in channel any text I see "TestReplyToAnyText" in that channel.
Tell me please how to make 
`$this->reply('Channel post content.....');`
to be sent exactly to my channel from Laravel ?

I mean I want to be able to send text from backend to channel with help of my bot

**Reply by D0nVitalio** [reply]
Found solution :)
```
$channel = TelegraphChat::find(2);
$channel->message('Hello from Laravel backend!')->send();
```

**Reply by fabio-ivona** [reply]
Glad you have found it! ❤️
